Application/EdFi.Ods.Admin.Api/E2E Tests/Admin API E2E.postman_collection.json

```diff
@@ -3962,26 +3929,22 @@
 									"listen": "test",
 									"script": {
 										"exec": [
-											"pm.test(\"Status code is Bad Request\", function () {\r",
-											"    pm.response.to.have.status(400);\r",
+											"pm.test(\"Status code is Not Found\", function () {\r",
+											"    pm.response.to.have.status(404);\r",
```

Comment:
Previously, this test was simply wrong: it is a test for a PUT request when the ID does not exist. I don't know why the test was checking for a 400 instead of a 404 status code.